src/test/resources/integration-test-mule-config.xml

```diff
@@ -28,4 +30,16 @@
         <akv:encrypt-key config-ref="config" keyName="${akv.test.key.name}" alg="${akv.test.alg}" value="${akv.test.value}" />
         <akv:decrypt-key config-ref="config" keyName="${akv.test.key.name}" alg="${akv.test.alg}" value="#[payload.value]" />
     </flow>
+
+    <!-- TLS Testing -->
+    <flow name="validTlsTest" >
```

Comment:
Is there a test for invalid since it would pass as valid without a trust store at all?

Reply:
No, I had an empty truststore, and would only get timeout errors....

Reply:
but, including the truststore config, at least tries making a connection.... so an invalid truststore would fail...

Reply:
I'd have some use case that shows it is at least trying to use a trust store, and failing, to prove its actually using the trust store that is conifgured.